src/bin/juliaup.rs

```diff
@@ -27,128 +29,6 @@ use juliaup::{
 };
 use log::info;
 
-#[derive(Parser)]
```

Comment:
Disclaim: I am not a maintainer of this repository, so feel free to ignore me. This advice is just based on my experience maintaining other projects. If this PR was one on my repository, I'd be reluctant to review and merge it as it is somewhat opaque.

Specifically: Do you *need* to move these definitions to a new file? If not, I would recommend against doing it, at least in this PR, as it makes it that much harder to review it, as it is not obvious if anything (and what) was changed in this code block other than moving it.


If the code must be split out, I would recommend doing it in two commits: the first commit just moves the code to a new file, but without otherwise modifying it. The second commit than does the actual work and modifications.

Reply:
Thanks for your feedback.
Unfortunately I think separating the files is necessary, otherwise the `command_completions.rs` cannot access the `enum Juliaup`.

I've now separated the commits like you recommended.